prisma/schema.prisma

```diff
@@ -40,18 +40,19 @@ model User {
 model Recipe {
   id               Int        @id @default(autoincrement())
   name             String
-  content          String
+  description      String
+  steps            String[]
+  ingredients      String[]
```

Comment:
hm now the question is how do we separate name from quantity?

Reply:
damn i forgot about it XD